src/components/themed/WalletListCurrencyRow.tsx

```diff
@@ -50,13 +55,40 @@ const WalletListCurrencyRowComponent = (props: Props) => {
 
   return (
     <TouchableOpacity accessible={false} style={styles.row} onLongPress={handleLongPress} onPress={handlePress}>
-      {customAsset != null ? <CustomAssetRow customAsset={customAsset} /> : <CurrencyRow showRate={showRate} token={token} tokenId={tokenId} wallet={wallet} />}
+      {isPaused ? (
+        <>
+          <View style={styles.overlayLabelContainer}>
+            <EdgeText style={styles.overlayLabel}>{lstrings.fragment_wallets_wallet_paused}</EdgeText>
+          </View>
+          <View style={styles.overlayContainer} />
+        </>
+      ) : null}
+      {customAsset != null ? (
+        <CustomAssetRow customAsset={customAsset} />
+      ) : (
+        <CurrencyRow showRate={showRate && !isPaused} token={token} tokenId={tokenId} wallet={wallet} />
+      )}
     </TouchableOpacity>
   )
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
-  // Layout:
+  overlayLabel: {
+    color: theme.overlayDisabledTextColor
+  },
+  overlayLabelContainer: {
+    ...StyleSheet.absoluteFillObject,
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    zIndex: 2
```

Comment:
Instead of using `zIndex`, just move the `isPaused ? <>..</> : null` block *after* the `<CurrencyRow>`rendering block. Then it will naturally go on top of the row component.